extensions/sql-bindings/src/common/azureFunctionsUtils.ts

```diff
@@ -307,3 +309,207 @@ export async function getAFProjectContainingFile(fileUri: vscode.Uri): Promise<v
 export async function isFunctionProject(folderPath: string): Promise<boolean> {
 	return fs.existsSync(path.join(folderPath, constants.hostFileName));
 }
+
+export async function promptForBindingType(): Promise<(vscode.QuickPickItem & { type: BindingType }) | undefined> {
+	const inputOutputItems: (vscode.QuickPickItem & { type: BindingType })[] = [
+		{
+			label: constants.input,
+			type: BindingType.input
+		},
+		{
+			label: constants.output,
+			type: BindingType.output
+		}
+	];
+
+	const selectedBinding = (await vscode.window.showQuickPick(inputOutputItems, {
+		canPickMany: false,
+		title: constants.selectBindingType,
+		ignoreFocusOut: true
+	}));
+
+	return selectedBinding;
+}
+
+export async function promptForObjectName(bindingType: BindingType): Promise<string | undefined> {
+	return vscode.window.showInputBox({
+		prompt: bindingType === BindingType.input ? constants.sqlTableOrViewToQuery : constants.sqlTableToUpsert,
+		placeHolder: constants.placeHolderObject,
+		validateInput: input => input ? undefined : constants.nameMustNotBeEmpty,
+		ignoreFocusOut: true
+	});
+}
+
+export async function promptForConnectionStringSetting(projectUri: vscode.Uri | undefined): Promise<string | undefined> {
```

Comment:
`promptAndUpdateConnectionStringSetting` - this isn't just prompting so we should be clear here